spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/KubernetesConfigDataResource.java

```diff
@@ -62,6 +62,20 @@ public KubernetesClientProperties getProperties() {
 		return this.properties;
 	}
 
+	/**
+	 * ConfigMapConfigProperties that might be null.
+	 */
+	public ConfigMapConfigProperties getConfigMapProperties() {
```

Comment:
They were here before, then I decided to remove them, since they were un-used and now, in order to properly test - I am adding them back